res/mac/FreenetTray/src/LICENSE

```diff
@@ -1,81 +1,45 @@
 Original code:
 
 This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+it under the terms of the GNU General Public License version 2, 
+as published by the Free Software Foundation.
```

Comment:
First, the license. It's not at all important to *me* which license is used. My concern and reasoning to change it is to protect FPI:

* In the near future, a binary of this app will very likely to be signed by an Apple developer certificate that FPI obtains
* GPLv3 has some terms that apply to code signing keys, etc (i.e. anti-tivoization). They explicitly **don't** apply to "proof of origin" signatures, but they **do** apply to "permission to run" signatures like these when the platform is enforcing them
* I'm not a lawyer and I can't even make a reasonable guess as to how GPLv3 will apply in this situation *right now*, let alone just a year from now, if for instance Apple made platform changes that bring Macs even closer to the TiVO/iOS behavior (i.e. no more kill switch on "permission to run" signature checks, exactly how iOS works. That's a real possibility). 
* So then where does that leave FPI.... do Apple's future platform security changes automatically affect FPIs obligations under GPLv3? Is there a forced choice between breaking the GPLv3 terms, or handing over a private signing key and having it get revoked by Apple? I honestly have no idea, and as a professional iOS and Mac developer, I would *never* put a client in this situation, I can't put FPI in such a situation either, if for no other reason than purely out of an abundance of caution

Second, the code itself. Virtually all of the modifications to my original code are separate ~3-4 line style changes or bugfixes, I'm not sure whether they fall under fair use or would be too short to copyright, but I don't like the idea of treating them that way or guessing about it. Most of them have already been completely replaced/deleted anyway due to natural obsolescence or syntax changes, what's left are 2-3 small changes in parts of the original code that I was already rewriting, so they won't be present at all once I push those changes.